@here/olp-sdk-dataservice-read/lib/client/QuadTreeIndexRequest.ts

```diff
@@ -106,6 +107,18 @@ export class QuadTreeIndexRequest {
         return this;
     }
 
+    /**
+     * A setter for the provided additional fields.
+     * 
+     * @param fields Array of strings. Array could contain next values "dataSize" | "checksum" | "compressedDataSize".
```

Comment:
Please add information about how that parameter could be useful

Reply:
Added